### DISCUSSION TITLE
Where can I set the default loading page ?

### DISCUSSION THREAD

**Discussion by 0xPufferFish** [SchabanBo/qlevar_router#141] [Q&A] [answered]
When I'm entering a page for the first time and the auth middleware is working, the app displays this screen with "Loading" text in the center, I didn't make that screen so it must be from the router right? If so where can I set this loading screen, thanks
![image](https://github.com/SchabanBo/qlevar_router/assets/96884227/4062654f-7070-4146-a14b-673b92f8e3c2)

**Comment by SchabanBo** [comment, marked as answer]
To change this page, you can do this
```dart
QR.settings.initPage = YourWidget();
```
